scripts/man-dates.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
This new script should be added to `EXTRA_DIST` in the `Makefile.am`.

Reply:
I'll admit that I initially didn't think of that at all. But considering it now... should it? The whole point of the script is to make it easy to fix the dates in the VCS (git). In other words, it's useful in a checkout, but not a tarball. That said, I added it per your request.

I also see that `zol2zfs-patch.sed` was missing, despite the opposite `zfs2zol-patch.sed` being present, so I fixed that. Then I sorted the list.

Reply:
That's a good point.  I suppose it really doesn't matter, but for better or worse we've been including all of these helper scripts in the tarballs.  At least for the moment  let's remain consistent.  If someone feels strongly about not including them we can revisit it and consider all of the EXTRA_DIST scripts (For example, commitcheck.sh).